ArcGISRuntimeSDKQt_CppSamples/Geometry/DensifyAndGeneralize/DensifyAndGeneralize.cpp

```diff
@@ -126,11 +126,12 @@ void DensifyAndGeneralize::updateGeometry(bool densify, double maxSegmentLength,
   m_resultLineGraphic->setGeometry(polyline);
 
   // Update the multipoint graphic
-  MultipointBuilder multipointBuilder(polyline.spatialReference());
-  PointCollection* pointCollection = new PointCollection(polyline.spatialReference(), this);
```

Comment:
This is not a leak, because `this` is the parent, but if the parts is empty, this function returns without deleting the point collection.